Sun_Shengran/progress.md

```diff
@@ -3,78 +3,71 @@
 **Project: Recycle Bin**
 
 Overall Direction:
-- 
-- 
-- 
+- Build a recovery page for instructors to display user deleted courses, sessions and students
+- Allow instructors to recover deleted courses, sessions and students from the recycle bin
+- Allow instructors to permanently remove selected/all items from the recycle bin
+- Build UI presentation for the deleted items and relevant functions
+- Auto remove items from the recycle bin after a certain period
+- Create comprehensive tests for the recovery feature
 
 ---
 
 Week 1 Plan:
-- [ ] 
-- [ ] 
-- [ ] 
+- [ ] Build UI frame/structure for displaying of deleted courses, sessions and students
+- [ ] Create fake deleted items data for the acceptance testing of UI
 
 Week 2 Plan:
-- [ ] 
-- [ ] 
-- [ ] 
+- [ ] Add new attribute "deletedDate" for courses, sessions and students
+- [ ] Modify relevant tests for courses, sessions and students
```

Comment:
Just to clarify, by "Modify relevant tests..." do you mean that you will add to existing tests to check the default value of the "deletedDate" attribute?

Reply:
For that week, yes it's just to check whether the attribute with default value has been added correctly. For the next week, those tests related to deletion need to be modified, because items are not supposed to be deleted since then.